R/utils.R

```diff
@@ -1,17 +1,17 @@
-library("dplyr")
-
-# function to return folder name
+#' function to return folder name
```

Comment:
This file has mostly been copied from global.R. The filepaths in `get_filename` have been modified to the get the app to find the correct files from the server and `get_all_clusters` has been refactored slightly. The latter function could be rewritten more succinctly without dplyr.

There's a bit of quirky code in `reformat_table` which I didn't follow, but since you didn't modify that function we'll leave it as is. I'm guessing it could be replaced with janitor::row_to_names (but might not have understood the code)